src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public abstract class Figure implements IAreaCalculator, IRandomSide {
+    private String name;

```

Comment:
remove this field, you don't need it if you want to add such value for each figure, add it to getDrawing directly